### PR TITLE
Finished issue 7515

### LINKED ISSUE #7515: [Docs Testing] Test all available commands from "AI Workflow -> Create Projects" section using SQL
The objective of this issue is to test all available commands from "AI Workflow -> Create Projects" section using SQL

## Resources

**Here is what should be tested in the MindsDB editor:**
https://docs.mindsdb.com/sql/create/project
https://docs.mindsdb.com/sql/drop/project
https://docs.mindsdb.com/sql/list-projects

## Steps to Follow

1. Test all the commands as outlined in the Resources above.
2. Save the test results, including commands and their outputs, in the `Testing.md` file in a new GitHub repository in your account.
3. Create a PR in the [MindsDB repository](https://github.com/mindsdb/mindsdb) following these steps:
	- Go to the `/docs/` folder and open the `Docs_Manual_QA.md` file.
	- If all the tests passed, append this line: `AI_WORKFLOW_SECTION (DEV_ENV) Test Passed: <link to the test results in your GitHub repository>`
	- If some tests failed, create an issue to report it and append this line: `AI_WORKFLOW_SECTION (DEV_ENV) Test Failed: <link to the test results in your GitHub repository>, <link to the issue>`

**Please note that we are not assigning this type of issue.
Multiple people can work on it and submit their results in PRs** :fireworks:

## :loudspeaker: Additional rewards :trophy:

Each integration test brings 10+ points for SWAG and entry into the draw for a :point_down: :
-  :computer:  [Razer Blade 16 Laptop](https://www.razer.com/gaming-laptops/Razer-Blade-16/RZ09-0483SEJ3-R3U1)
- :shirt: :bear:  [MindsDB Swag](https://store.covver.io/mindsdb/collections/swag-collection)

:information_source: For more info check out https://mindsdb.com/hacktoberfest/ :point_left: 

**Please note that we are not assigning this type of issue.
Multiple people can work on it and submit their results in PRs** :fireworks:

### PR DESCRIPTION
**Pull request by hridaya423**
## Description

Tested all the commands in docs of issue 7515

Fixes #7515 

## Type of change
- [x] 📄 This change requires a documentation update

## Verification Process

To ensure the changes are working as expected:

 - [x]   Test Location: Specify the URL or path for testing.
 - [x]   Verification Steps: Outline the steps or queries needed to validate the change. Include any data, configurations, or actions required to reproduce or see the new functionality.

## Additional Media:

- [x] I have attached a brief loom video or screenshots showcasing the new functionality or change.

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [x] I have appropriately commented on my code, especially in complex areas.
- [x] Necessary documentation updates are either made or tracked in issues.
- [x] Relevant unit and integration tests are updated or added.

### CODE REVIEW

**Comment by chandrevdw31** [comment]
@hridaya423 please update your doc with images and also test List Projects without the brackets `SHOW FULL DATABASES
WHERE type = 'project';`

File: docs/Docs_Manual_QA.md [hunk 1]
```diff
@@ -13,3 +13,7 @@ XXX Test Passed: <link to the test results in your GitHub repository>
 </br>
 XXX Test Failed: <link to the test results in your GitHub repository>, <link to the issue>
 </br>
+
+Create a Project, Test passed: https://github.com/hridaya423/HacktoberfestMindsDBTesting/blob/master/Testing.md
+Remove a Project, Test passed: https://github.com/hridaya423/HacktoberfestMindsDBTesting/blob/master/Testing.md
+List Projects, Test Failed: https://github.com/hridaya423/HacktoberfestMindsDBTesting/blob/master/Testing.md, https://github.com/mindsdb/mindsdb/issues/7519
```
Comment (chandrevdw31) [inline docs/Docs_Manual_QA.md:19]: Test with syntax `SHOW FULL DATABASE WHERE type = 'project';` without brackets please

Comment (chandrevdw31) [inline docs/Docs_Manual_QA.md:19]: Please update the file by removing test failed and the github issue and add that the test passed

Comment (hridaya423) [reply docs/Docs_Manual_QA.md:19]: How do i edit it?

**Comment by hridaya423** [comment]
@chandrevdw31  I updated it

**Review by chandrevdw31 — APPROVED** [review]

**Pull request merged**